### PR TITLE
Align Chart and holders - Collection Activity

### LINKED ISSUE #7853: Align chart and holders section in one line?
Would it be possible to have the upper borders of "All" and "Holders" being aligned? 

https://kodadot.xyz/ahk/collection/u-2022/activity

image

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #7853

## Screenshot 📸

- [x] My fix has changed UI

image

### <samp>🤖 Generated by Copilot at 1e2694d</samp>

Fixed tooltip alignment in price chart component. Removed `top` property from `.price-chart__tooltip` in `_price-chart.scss`.

### <samp>🤖 Generated by Copilot at 1e2694d</samp>

> _`top` property gone_
> _tooltip aligns with chart_
> _a winter bug fixed_

### CODE REVIEW

**Comment by exezbcz** [comment]
yes, looks cool, but now the divider is right on the button
image

**Comment by yangwao** [comment]
> now the divider is right on the button

true

image

yet dropdown seems bit off? I know even in older version was

image

**Review by Jarsen136 — APPROVED** [review]

File: components/collection/activity/Activity.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <div ref="wrapper" class="w-full mt-5">
       <div v-if="tablet">
         <div class="columns">
-          <div class="column is-two-thirds">
+          <div class="column is-two-thirds mt-2">
```
Comment (roiLeo) [inline components/collection/activity/Activity.vue:7]: 😕 margin on a column is a bad idea

**Review by roiLeo — COMMENTED** [review]
It looks worse

Better solution would be to change aboslute position on dropdown

File: components/collection/activity/Activity.vue (hunk 1 above)
Comment (daiagi) [reply components/collection/activity/Activity.vue:7]: 1. why?
2. 
>Better solution would be to change aboslute position on dropdown 
yes, did that https://github.com/kodadot/nft-gallery/pull/7859/commits/1e2694d1f495bcb20e681b316bac862131f88360
and...

https://github.com/kodadot/nft-gallery/pull/7859#issuecomment-1783738246

Comment (roiLeo) [reply components/collection/activity/Activity.vue:7]: please could you move `mt-2` to the content (`ActivityChart` component) so we wouldn't have futur side effects

Comment (daiagi) [reply components/collection/activity/Activity.vue]: done

**Review by roiLeo — APPROVED** [review]

**Pull request merged**